src/xb-node.c

```diff
@@ -49,6 +49,32 @@ typedef struct
 G_STATIC_ASSERT (sizeof (XbNodeAttrIter) == sizeof (RealAttrIter));
 G_STATIC_ASSERT (G_ALIGNOF (XbNodeAttrIter) >= G_ALIGNOF (RealAttrIter));
 
+/**
+ * XbNodeChildIter:
+ *
+ * A #XbNodeChildIter structure represents an iterator that can be used
+ * to iterate over the children of a #XbNode. #XbNodeChildIter
+ * structures are typically allocated on the stack and then initialized
+ * with xb_node_child_iter_init().
+ *
+ * The iteration order of a #XbNodeChildIter is not defined.
```

Comment:
Is it correct that the iteration order is not defined? Unlike attributes, the child nodes in an XML element tree are ordered, and order does matter in some XML schemas (like HTML).

Reply:
True - the XbSilo keeps the order, and we iterate in-order, so that comment isn't actually correct.